src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -230,20 +245,15 @@ protected Task Stop()
                     }
                 });
         }
-
         protected async Task HandleShowOnlineHelpRequest(
             string helpParams,
             RequestContext<object> requestContext)
         {
-            if (helpParams == null) { helpParams = "get-help"; }
-
-            var psCommand = new PSCommand();
-            psCommand.AddCommand("Get-Help");
-            psCommand.AddArgument(helpParams);
-            psCommand.AddParameter("Online");
-
-            await editorSession.PowerShellContext.ExecuteCommand<object>(psCommand);
+            if (helpParams == null) { helpParams = "Get-Help"; }
 
+            PSCommand checkHelpPSCommand = new PSCommand();
```

Comment:
This command instantiation could probably be streamlined a bit into:
```csharp
var checkHelpPSCommand = new PSCommand()
    .AddScript(CheckHelpScript, useLocalScope: true)
    .AddArgument(helpParams);
```

Reply:
chain all the things?